src/parser/parser.yy

```diff
@@ -883,7 +853,49 @@ edge_prop_expression
 
 function_call_expression
     : LABEL L_PAREN opt_argument_list R_PAREN {
-        $$ = new FunctionCallExpression($1, $3);
+        if (!$3) {
+            $$ = new FunctionCallExpression($1, $3);
+        } else if ($3->numArgs() == 1 && AggFunctionManager::find(*$1).ok()) {
```

Comment:
Function names should not be case-sensitive. the function name in FunctionManager with some uppercase and some lowercase. need to unify it. Otherwise, the `find` will be incorrect.

Reply:
Already handled in `AggFunctionManager::getInternal`.

Reply:
Currently, functions are case sensitive.